OracleDatabase/dockerfiles/buildDockerImage.sh

```diff
@@ -103,7 +111,7 @@ elif [ $EXPRESS -eq 1 ] && [ "$VERSION" != "11.2.0.2" ]; then
   exit 1;
 else
   EDITION="xe";
-  DOCKEROPS="--shm-size=1G";
+  DOCKEROPS="--shm-size=1G $DOCKEROPS";
```

Comment:
No, this is done so that I won't loose other options passed on. The options are set between line 88 and 94 of the script.